### PR TITLE
Good Bye, youtube-dl!

### PR DESCRIPTION
**Pull request by ycanardeau**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,5 @@ VocaDbWeb.Resources/obj/
 *.mdf
 *.ldf
 *.ndf
+VocaDb.VideoInfoExtractors/bin/
```
Comment (riipah) [inline .gitignore:36]: Is this necessary anymore?

File: VocaDbModel/Service/VideoServices/VideoServiceBandcamp.cs [hunk 2]
```diff
@@ -33,16 +32,7 @@ public class VideoServiceBandcamp : VideoService {
...
-				RetrieveAllInfo = true,
-				YoutubeDlPath = GetPath(AppConfig.YoutubeDLPath),
-				PythonPath = GetPath(AppConfig.PythonPath)
-			};
-
-			youtubeDl.StandardOutputEvent += StandardOutputEvent;
-			youtubeDl.StandardErrorEvent += StandardErrorEvent;
-
-			DownloadInfo result;
+			/*DownloadInfo result;
```
Comment (riipah) [inline VocaDbModel/Service/VideoServices/VideoServiceBandcamp.cs:35]: I think you can just delete the commented code. It's in github history anyway.

**Comment by riipah** [comment]
Thanks!

**Comment by ycanardeau** [comment]
Thanks for your review!

**Pull request merged**